docs/editors/new-editor.md

```diff
@@ -275,6 +280,13 @@ Possible values:
   For example, `C:\Users\IEUser\workspace\project/*.{scala,sbt,properties}`.
   This mode is used by the VS Code client.
 
+##### `inlineDecorationProvider`
+
+If the client implements the Metals Decoration Protocol **and** supports decorations
+to be show inside text and not only at the end of a line.
```

Comment:
unsure about it but shouldn't it be `to be shown`?

Reply:
You're absolutely right. It was a shameless copy pasta from `InitializationOptions`. I've update this and also the scaladocs.